src/test/kotlin/org/opensearch/replication/integ/rest/ResumeReplicationIT.kt

```diff
@@ -210,6 +210,14 @@ class ResumeReplicationIT: MultiClusterRestTestCase() {
     }
 
     fun `test that replication fails to resume when custom analyser is not present in follower`() {
+
+        //check if integTest remote = true
```

Comment:
DId you also explore having this logic as an annotation like we had discussed?

Reply:
The logic similar to @Awaitsfix is in Apache Lucene library 
 public static final String SYSPROP_AWAITSFIX = "tests.awaitsfix";

We will not be able to ignore tests without modifying the project.

Reply:
`MultiClusterRestTestCase` does extends `LuceneTestCase` implicitly.  I think you should be able to add an annotation interface which is enabled by default but gets disabled if `tests.integTestRemote` is set to true.


If it doesn't work, then we can also explore moving this common logic as a method in `MultiClusterRestTestCase`
and simply keep it as `if(isRemoteIntegTest()) return;`